doc/docs/usage/quickstart.md

```diff
@@ -22,38 +24,33 @@ workloads:
         value: Ankaios team
     runtimeConfig: |
       image: docker.io/nginx:latest
-      ports:
-      - containerPort: 80
-        hostPort: 8081
+      commandOptions: ["-p", "8081:80"]
 ```
 
 1.  Note that access rights are currently not implemented.
 
 
-Before we start Ankaios we need to make sure that Podman is listening on a
-socket that can be used by Ankaios.
+Then we can start the Ankaios server:
 
 ```shell
-systemctl --user start podman.socket
+systemctl start ank-server
```

Comment:
Personally I get an error without sudo. Is this missing here or do you expect I am already logged in as root? Same for the agent below.

```suggestion
systemctl start ank-server
```

Reply:
OK. I will add `sudo`. On Manjaro you get a popup window asking for the root password.